.github/build.yml

```diff
@@ -0,0 +1,17 @@
+name: Build
```

Comment:
This should be inside `.github/workflows/`. CMIIW

Reply:
correct. fixed it now.